sycl/source/detail/event_impl.hpp

```diff
@@ -215,10 +215,11 @@ class event_impl {
   void instrumentationEpilog(void *TelementryEvent, const std::string &Name,
                              int32_t StreamID, uint64_t IId) const;
 
-  RT::PiEvent MEvent = nullptr;
-  ContextImplPtr MContext;
-  bool MOpenCLInterop = false;
-  bool MHostEvent = true;
+  mutable bool isInited = true;
```

Comment:
The naming of this is not consistent with the other members. It should instead be `MIsInited` or personally I would prefer `MIsInitialized`

Reply:
Thanks, I will fix it in the nearest patch, sorry for the issue.